django_lightweight_queue/urls.py

```diff
@@ -1,9 +1,9 @@
-from django.conf.urls import url
+from django.urls import re_path
```

Comment:
```suggestion
from django.urls import path
```
(for the other suggestion)